app/javascript/components/supplemental_application/sections/ConfirmedHouseholdIncome.js

```diff
@@ -51,22 +51,23 @@ const ConfirmedHouseholdIncome = ({ listingAmiCharts, form, visited }) => {
 
   useEffect(() => {
     const getAmiCharts = async () => {
-      let chartTypes = []
       let amis = await getAmis(listingAmiCharts, totalHouseholdSize)
```

Comment:
we should be able to remove this query entirely depending on what's available in listingAmiCharts already.

Reply:
I pushed a removal here and found that the getAmis was only used here and in a file testing the function so I removed those instances as well